lib/Driver/DarwinToolChains.cpp

```diff
@@ -367,6 +367,18 @@ toolchains::Darwin::addArgsToLinkStdlib(ArgStringList &Arguments,
         Arguments.push_back(context.Args.MakeArgString(BackDeployLib));
       }
     }
+
+    if (*runtimeCompatibilityVersion <= llvm::VersionTuple(5, 1)) {
```

Comment:
There's another version check slightly above this for the `runtime-compatibility-version` flag. Should that be updated as well?

Reply:
Indeed, thanks!